hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -101,8 +113,10 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
     });
     axios.all([
       axios.get('/api/v1/clusterState'),
-      axios.get('/api/v1/task/status')
-    ]).then(axios.spread((clusterStateResponse, taskstatusResponse) => {
+      axios.get('/api/v1/task/status'),
+      axios.get('/api/v1/keys/open?limit=0'),
+      axios.get('/api/v1/keys/deletePending?limit=1'),
```

Comment:
check if limit=1 or limit=1 is correctly used...

Reply:
@sumitagrawl 
Yes checked and discussed with @ArafatKhan2198  for deletePending  need to set limit 1 to fetch only 1 record with summary data.